packages/relay/src/lib/eth.ts

```diff
@@ -188,13 +188,14 @@ export class EthImpl implements Eth {
         return EthImpl.feeHistoryZeroBlockCountResponse;
       }
 
-      let feeHistory: object | undefined = this.cache.get(constants.CACHE_KEY.FEE_HISTORY);
+      const cacheKey = `${constants.CACHE_KEY.FEE_HISTORY}_${blockCount}_${newestBlock}_${rewardPercentiles?.join('')}`;
```

Comment:
Do we need all the params?

Reply:
Yep, in any other case the returned response will not be valid or fulfilled :confused: